client/src/main/java/io/dapr/DaprProtos.java

```diff
@@ -20,12 +20,10 @@ public static void registerAllExtensions(
 
     /**
      * <code>.google.protobuf.Any data = 1;</code>
-     * @return Whether the data field is set.
```

Comment:
Why did the auto-generated code remove the @return documentation?

Reply:
No clue. Need to dig into the Java generator. May be able to look late this week.